src/persist-plugins/mmkv.ts

```diff
@@ -85,7 +85,11 @@ export class ObservablePersistMMKV implements ObservablePersistPlugin {
         const v = this.data[table];
         if (v !== undefined) {
             try {
-                storage.set(table, safeStringify(v));
+                if (v !== null) {
+                    storage.set(table, safeStringify(v));
```

Comment:
On second thought, I think the root issue is how `safeStringify()` is implemented:

```
return value ? JSON.stringify(value, replacer) : value;
```

It means that if the value is false (`false`, `0`, `""` or `null`), it won't be serialized as a string. If this called `JSON.stringify()` directly, this would work properly as `null` would be serialised `"null"` which can be saved and restored with MMKV.

I'm not sure I understand the intent of `safeStringify()`, I would expect it to always return either a string or undefined.

Reply:
It probably also means that persisting `0` (or `false`) is broken too, as it will be persisted as a number but we will attempt to restore it with `storage.getString()`